tools/behavior_simulator/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod cycler;
```

Comment:
Yes, twix currently depends on the cycler databases.
However, that is a good point. Currently the behavior cycler database is identical to the control cycler database since robots have no state that isn't in the control cycler database anyway.
I planned on keeping things like robot velocity or progress in currently playing motion in there. At the moment, kick motions for example are executed instantaneously.
For now we could just use the control cycler database, what do you think?

Reply:
Removed the duplicate database type. Now using `cyclers::control::Database`.